src/BaGet/Controllers/PackagePublishController.cs

```diff
@@ -77,6 +76,7 @@ public async Task Upload(CancellationToken cancellationToken)
             }
         }
 
+        [HttpDelete]
```

Comment:
Weird. This worked in 2.1 by adding constraints like `HttpMethodRouteConstraint("POST")`. I wonder if registering the route with `MapPost` and `MapDelete` [here](https://github.com/loic-sharma/BaGet/blob/master/src/BaGet/Extensions/IRouteBuilderExtensions.cs#L25) would fix this issue.

Reply:
`MapPost` and `MapDelete` don't accept a name, which would break the URL generation. So that'd be more work to fix